enumeratum/src/main/scala/com/monovore/decline/enumeratum/package.scala

```diff
@@ -5,16 +5,23 @@ import cats.data.{Validated, ValidatedNel}
 import _root_.enumeratum._
 import _root_.enumeratum.values._
 
+import scala.collection.immutable.IndexedSeq
+
 package object enumeratum {
 
+  private[enumeratum] def invalidChoice(missing: String, choices: IndexedSeq[String]): String =
+    s"Invalid choice provided ($missing), choose one from <${choices.mkString(", ")}>"
```

Comment:
This is stylistically a little bit out of line with the other error messages from this library; perhaps:

```suggestion
    choices match {
      case Seq() => s"Invalid value $missing"
      case Seq(value) => s"Invalid value $missing; expected $value"
      case many => s"Invalid value $missing; expected ${many.init.mkString(", ")} or ${many.last}"
    }      
```